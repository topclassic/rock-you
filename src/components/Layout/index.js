import React from "react";
import styled from "styled-components";

const Main = styled.div`
  min-height: 100vh;
  display: flex;
  align-items: center;
  justify-content: center;
`;

const CalContainer = props => {
  const { children } = props;
  return <Main>{children}</Main>;
};

export default CalContainer;
