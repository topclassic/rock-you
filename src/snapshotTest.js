
import renderer from 'react-test-renderer'

const snapshotTest = (testCase, element) => {
  it(testCase, () => {
    const wrapper = renderer.create(element).toJSON()
    expect(wrapper).toMatchSnapshot()
  })
}

export default snapshotTest