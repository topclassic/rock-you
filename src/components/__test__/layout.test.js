import React from "react";
import { shallow } from "enzyme";
import Layout from "../Layout";
import snapshotTest from "../../snapshotTest";

describe("Layout", () => {
  it("test render component correctly", () => {
    const wrapper = shallow(<Layout />);
    expect(wrapper).toHaveLength(1);
  });
});

describe("Layout testing", () => {
  snapshotTest("render Layout", <Layout />);
});
