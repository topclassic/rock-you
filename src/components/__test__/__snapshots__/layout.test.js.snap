// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout testing render Layout 1`] = `
<div
  className="sc-bdVaJa kNCorC"
/>
`;
